feat: replace chat history in Analytics with chat dialogs; split chat message component

alexdorowork opened the pull request.

lizozom commented:
When you click on `Show References` in the history view, it's behaving weird.

![image](https://github.com/user-attachments/assets/ffbb8e94-ca64-4314-b6e0-5e1065ae694f)

lizozom commented:
![image](https://github.com/user-attachments/assets/9dcbf59e-a0f5-4c96-a8e6-5c2a1a4babfa)
All messages show "Unknown flow" for me

lizozom commented:
![image](https://github.com/user-attachments/assets/b510af44-0e32-47eb-8c30-9317c11bc285)

Maps appear broken in view

lizozom commented:
Add the user email to each displayed item (maybe smaller, gray font under the title?)

alexdorowork commented:
> ![image](https://private-user-images.githubusercontent.com/3016806/452761580-b510af44-0e32-47eb-8c30-9317c11bc285.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDkzODY5NDUsIm5iZiI6MTc0OTM4NjY0NSwicGF0aCI6Ii8zMDE2ODA2LzQ1Mjc2MTU4MC1iNTEwYWY0NC0wZTMyLTQ3ZWItOGMzMC05MzE3YzExYmMyODUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDYwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA2MDhUMTI0NDA1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NzM2MWE1ZmNkOTAzYTQxYzEzODdmN2YzMDIxZDMwNTAzNDcwMjhmZmZkZDQ0NDE1MTg5ZGQ1NDJkYmJkMWFjMSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.lhWqS-WE9vgJXsuhHV03OIxi1mTJf5Cxr9F7DrZFuJQ)
> 
> Maps appear broken in view

Do you have any errors in browser console when it happens?
Just tried to open the same dialog and the map looks well

alexdorowork commented:
@lizozom 
Regarding "Unknown flow":
Do you have ui.default_flow in config?

alexdorowork commented:
@lizozom updated:

1) https://github.com/ShragaAI/shraga-ui/pull/24#pullrequestreview-2908314862
Replaced popover window with popup in the center


2) https://github.com/ShragaAI/shraga-ui/pull/24#issuecomment-2953860210
fixed

3) https://github.com/ShragaAI/shraga-ui/pull/24#issuecomment-2953860434
It works well with config for production.
It's necessary to have ui.list_flows and ui.default_flow in config. If we need to do them not mandatory, so we have to change some logic in FE.

4) https://github.com/ShragaAI/shraga-ui/pull/24#issuecomment-2953861473
It works well with config for production.
Be aware that you use the correct Google Maps API key.

5) https://github.com/ShragaAI/shraga-ui/pull/24#issuecomment-2953862690
Added
<img width="838" alt="image" src="https://github.com/user-attachments/assets/65db8cd1-1b15-4451-9928-95e531ad7887" />
